src/libs/runtime-utils.js

```diff
@@ -10,17 +10,21 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
```

Comment:
Thought about renaming this file as `compute-utils` but it was going to increase the diff's even more. Could still do it if nudged 😉